README.md

```diff
@@ -304,6 +304,8 @@ pods = client.get_pods as: :raw
 node = client.get_node "127.0.0.1", as: :raw
 ```
 
+Other formats are: `:parsed` for `JSON.parse` or `:parsed_symbolized` for `JSON.parse(..., symbolize_names: true)`
+
```

Comment:
Could you include ```:ros``` also in the supported formats ?

Before this PR, any type other than ```:raw``` was considered as ```:ros```(default response format), or even if miss-typed (couldn't find any validation for the expected ```:as``` value). 

However, with this PR, it is mandatory now for clients to specify exactly ```:ros``` or not to pass a format in order to get response it in that format.

Reply:
added ros